### PR TITLE
Test branch: Nano enabled on master

### PR DESCRIPTION
**Pull request by alastairong**
This is a test branch to see exactly what commits need to be brought from develop to master to enable Nanos.

### CODE REVIEW

**Comment by alastairong** [comment]
Note the typo in the cherry pick leading to the commit called “t”. This is actually a cherry-pick of commit 1baa360f39e33a2125184c5f7f78b5936dc4e6b2

**Comment by alastairong** [comment]
Confirmed to work fine on my Nano.

File: profiles/physical/hpos/holoport-nano/default.nix [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
   boot.loader.grub.enable = false;
 
-  hardware.deviceTree.package = pkgs.holoport-nano-dtb;
+  services.holochain-conductor.enable = lib.mkForce false;
```
Comment (peeech) [inline profiles/physical/hpos/holoport-nano/default.nix:29]: I prolly missed it in the communications, but I have a question. We are disabling Holochain-conductor on Nano for now. What is the plan in the future for enabling it?

Comment (alastairong) [reply profiles/physical/hpos/holoport-nano/default.nix:29]: Probably with RSM

Comment (peeech) [reply profiles/physical/hpos/holoport-nano/default.nix:29]: ok

**Comment by peeech** [comment]
Obsolete in presence of #574

**Pull request closed without merging**